projects/SynthMemoryUsage/main.cpp

```diff
@@ -0,0 +1,132 @@
+#include "fp_util/feature_cmd.h"
+#include "fp_util/sleep.h"
+#include "fp_util/memory.h"
+
+#include <string>
+
+int main(int argc, char *argv[]) {
+  using fp_util::get_memory;
+  using fp_util::free_memory;
+  using fp_util::use_memory;
+
+  bool __attribute__((feature_variable("Slow"))) Slow = false;
+  bool __attribute__((feature_variable("PreAlloc"))) PreAlloc = false;
+  bool __attribute__((feature_variable("Header"))) Header = false;
+  bool __attribute__((feature_variable("Feature4"))) Feature4 = false;
+
+  if (fp_util::isFeatureEnabled(argc, argv, std::string("--slow"))) {
+    Slow = true;
+  }
+
+  if (fp_util::isFeatureEnabled(argc, argv, std::string("--prealloc"))) {
+    PreAlloc = true;
+  }
+
+  if (fp_util::isFeatureEnabled(argc, argv, std::string("--header"))) {
```

Comment:
```suggestion
  if (fp_util::isFeatureEnabled(argc, argv, std::string("--header"))) {
```
What's the idea behind the naming of this feature? If it's just a copy paste from the other one, I would suggest to change the name to something benchmark related. The name before referred to the fact that this feature variable/feature was implemented in a header file.